dpnp/dpnp_array.py

```diff
@@ -841,39 +841,37 @@ def flatten(self, order="C"):
 
         Parameters
         ----------
-        order: {'C', 'F', 'A', 'K'}, optional
-            'C' means to flatten in row-major (C-style) order.
-            'F' means to flatten in column-major (Fortran- style) order.
-            'A' means to flatten in column-major order if a is Fortran contiguous in memory, row-major order otherwise.
-            'K' means to flatten a in the order the elements occur in memory. The default is 'C'.
+        order : {"C", "F"}, optional
+            Read the elements using this index order, and place the elements
+            into the reshaped array using this index order. ``"C"`` means to
+            read / write the elements using C-like index order, with the last
+            axis index changing fastest, back to the first axis index changing
+            slowest. ``"F"`` means to read / write the elements using
+            Fortran-like index order, with the first index changing fastest,
+            and the last index changing slowest.
```

Comment:
I believe it would be much more readable to have it as a list:

```suggestion
            Read the elements using this index order, and place the elements
            into the reshaped array using this index order.

                - "C" means to read / write the elements using C-like index
                  order, with the last axis index changing fastest, back to the
                  first axis index changing slowest.
                - "F" means to read / write the elements using Fortran-like
                  index order, with the first index changing fastest, and the
                  last index changing slowest.

            The default is ``"C"``.
```